src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -8591,7 +8923,7 @@ namespace HVACUnitarySystem {
 				}}
 
 				//     IF outlet temp at no load is within ACC of set point, do not run the coil
-				if ( std::abs( Node( OutletNode ).Temp - DesOutTemp ) < Acc ) {
+				if ( std::abs( Node( OutletNode ).Temp - DesOutTemp ) < Acc  || UnitarySystem( UnitarySysNum ).HeatingCoilType_Num == Coil_UserDefined) {
```

Comment:
There is no need to call the user defined coil again since it will perform the same way. Just skip the full load test and move on.